metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -231,7 +231,7 @@ def usage_statistics_query(top_n: int) -> str:
                     ROW_NUMBER() over (PARTITION BY r.timestamp, r.resource, q.query ORDER BY COUNT(r.key) DESC, q.query) as rank
                 FROM
                     read_events r
-                    LEFT JOIN query_events q ON r.name = q.key
+                    INNER JOIN query_events q ON r.name = q.key
```

Comment:
It is possible to have a read event even if corresponding query event is missing.

https://cloud.google.com/bigquery/docs/reference/auditlogs/rest/Shared.Types/BigQueryAuditMetadata.TableDataRead.Reason